hash.c

```diff
@@ -6598,13 +6598,15 @@ env_dup(VALUE obj)
  *
  *  === Creating a \Hash
  *
- *  Here are three ways to create a \Hash:
+ *  You can create a \Hash object explicitly with:
  *
- *  - \Method <tt>Hash.new</tt>
- *  - \Method <tt>Hash[]</tt>
- *  - Literal form: <tt>{}</tt>.
+ *  - Global method {Hash}[Kernel.html#method-i-Hash].
+ *  - A {hash literal}[doc/syntax/literals_rdoc.html#label-Hash+Literals].
  *
- *  ---
+ *  Many Ruby methods return \Hash objects, including:
+ *
+ *  - Instance method +to_i+, available in some Core and Standard Library classes.
+ *  - Instance method +to_int+, available in fewer classes.
```

Comment:
```suggestion
 *  - Instance method +to_h+, available in some Core and Standard Library classes.
 *  - Instance method +to_hash+, available in fewer classes.
```

Reply:
Done, and thanks @esparta.  These are horrible errors.